pkg/deploy/taskdir/definitions/fixtures/sql.task.yaml

```
@@ -52,6 +52,10 @@ sql:
   # queryArgs:
   #   name: "{{params.name}}"
 
+  # The transaction mode to use. Valid values: auto, readOnly, readWrite, none.
+  # Default: auto.
+  transactionMode: auto
```

In this case, we would also comment this out like the other optional fields:

```suggestion
  # transactionMode: auto
```